gocd-ui-tweaks.user.js

```diff
@@ -0,0 +1,68 @@
+// ==UserScript==
+// @name         gocd-ui-tweaks
+// @namespace    https://deploy.getsentry.net
+// @version      0.1
+// @description  Minor tweaks to GoCD UI
+// @author       matt.gaunt
+// @match        https://deploy*.getsentry.net/**
+// @icon         https://www.google.com/s2/favicons?sz=64&domain=www.gocd.org
+// @grant        none
+// ==/UserScript==
+
+// Remove the agent links since agents are short lived and are generally 404.
+function dropAgentLink() {
+  const anchors = document.querySelectorAll('.index__agent-cell___1gGiw a');
```

Comment:
I suspect this class isn't going to be consistent -- any way we can genericize this?

Reply:
I found an ID further up the tree that should make more generic selectors safe to use.